tests/test_project.py

```diff
@@ -871,6 +871,106 @@ def test_update_submodules_strategy(repos_tmpdir):
     assert net_tools_project.sha('HEAD', cwd=kconfiglib_dst_dir) \
            == kconfiglib_new_sha
 
+@pytest.mark.xfail
+def test_update_submodules_relpath(tmpdir):
+    # Regression test for
+    # https://github.com/zephyrproject-rtos/west/issues/545.
+    #
+    # We need to make sure that submodules with relative paths are
+    # initialized correctly even when ther is no "origin" remote.
+    #
+    # Background: unless the config variable git.$SUBMODULENAME.url is
+    # set, git relies on the "default remote" for figuring out how to
```

Comment:
Very briefly mention that this is what `git submodule init` does: initializing this variable from the .gitmodules file.